src/targets/browser/index.jsx

```diff
@@ -39,7 +40,13 @@ const init = () => {
           <MuiCozyTheme>
             <BreakpointsProvider>
               <StepperDialogProvider>
-                <App />
+                <SharingProvider
+                  client={client}
+                  doctype="io.cozy.files"
+                  documentType="Files"
+                >
```

Comment:
Should be in the previous commit. Don't forget to add the CSS file

Reply:
Oh in fact, I'm wondering if the Provider should not be called by the FileViewer itself since this component should be standalone

Reply:
- J'ai revu mes commits :)
- L'import du css est dans le fichier où ce trouve le bouton Sharing (https://github.com/cozy/mespapiers/blob/0ee5ecdc84dcd9ebef4fafcef8db264509139629/src/components/Viewer/Footer/Sharing.jsx#L1)
- En effet, j'ai déplacé le composant `SharingProvider` dans le composant `FilesViewerWithQuery`